feature/gribi/otg_tests/static_lsp/README.md

```diff
@@ -0,0 +1,25 @@
+# TE-9.1: MPLS based forwarding Static LSP
+
+## Summary
+
+Validate static lsp functionality.
+
+## Procedure
+
+*  Create topology ATE1–DUT1-ATE2
+*  Enable MPLS forwarding and create egress static LSP to pop the label and forward to ATE2:
+*  Match incoming label (1000001)
+*  Set IP next-hop
+*  Set egress interface
+*  Set the action to pop label
+*  Start 2 traffic flows with specified MPLS tags IPv4-MPLS[1000002]-MPLS[1000001]
```

Comment:
I see only flow one flow w/ 2xlabels. Is that what you meant? So I think the packet will look something like below

<Ethernet><MPLS Label: 1000001><MPLS Label: 1000295><IP><TCP/UDP><DATA>

and the POP action is expected to pop `MPLS Label: 1000001`?

Reply:
oh! so I see the following. May be the steps need to be updated

Start 2 traffic flows with specified MPLS tags
IPv4-MPLS[1000001]
IPv4-MPLS[1000002]-MPLS[1000001]

Reply:
line 16 clarifies that traffic shud be received on dest ATE port with the outer label is removed.